### PR TITLE
Run MP FT 4.0 TCK with just core EE features

### PR DESCRIPTION
**Pull request by Azquelt**
For #22348

### CODE REVIEW

**Comment by Azquelt** [comment]
#libby
#build

**Review by tevans78 — APPROVED** [review]

File: dev/io.openliberty.microprofile.faulttolerance.4.0.internal_fat_tck/bnd.bnd [hunk 1]
```diff
@@ -23,7 +23,7 @@ tested.features:\
   expressionlanguage-5.0,\
   mpmetrics-5.0,\
   jsonp-2.1,\
-  servlet-6.0
+  restfulws-3.1
```
Comment (jhanders34) [inline dev/io.openliberty.microprofile.faulttolerance.4.0.internal_fat_tck/bnd.bnd:22]: you should be able to remove appsecurity-5.0, cdi-4.0 and expressionlanguage-5.0 with the update to the server.xml.

Comment (Azquelt) [reply dev/io.openliberty.microprofile.faulttolerance.4.0.internal_fat_tck/bnd.bnd:22]: Done, thank you.

**Comment by Azquelt** [comment]
Code review updates done, rebased on integration.
#libby
#build

**Comment by Azquelt** [comment]
An infrastructure issue caused the build to fail but all tests ran and passed.

**Pull request merged**